packs/actions/confident-finisher.json

```diff
@@ -16,7 +16,40 @@
         "requirements": {
             "value": ""
         },
-        "rules": [],
+        "rules": [
+            {
+              "key": "Note",
+              "selector": "attack",
+              "text": "You deal half your [[/r (2 + floor((@actor.level - 1)/4))d6/2]]{Precise Strike} damage to the target. This damage type is that of the weapon or unarmed attack you used for the Strike.",
```

Comment:
We should localize this, but also with the new suboptions I want to revisit how to handle the finishers I think. Confident finisher is only applied if you use that action, it doesn't apply on every finisher.